packs/salt/actions/lib/base.py

```diff
@@ -1,9 +1,23 @@
+# pylint: disable=no-member
+
 from st2actions.runners.pythonrunner import Action
 from requests import Request
+import logging
```

Comment:
Should he be using the `st2common` helper methods for this? https://github.com/StackStorm/st2/tree/master/st2common/st2common/logging